packages/template-master-detail-kinvey-ng/src/app/cars/car-detail-edit/_car-detail-edit.component.scss

```diff
@@ -20,6 +20,11 @@
     &__value {
         width: 65;
         text-align: right;
-        @include colorize($color: accent);
+        @include colorize($color: complementary);
+    }
+
+    Slider {
+        @include colorize($background-color: complementary);
```

Comment:
Don't use separate calls for these, as they generate two separate CSS rules that way. Use like this:

```scss
@include colorize(
    $background-color: complementary,
    $color: complementary
);
```

Reply:
Thanks!